crowdin/messages.json

```diff
@@ -37,13 +37,16 @@
   "34060478": "For forex, a deal for 1 lot of EUR/USD will pay out EUR 2.5 in commission based on the following formula:",
   "34997376": "Deriv in numbers",
   "35374636": "user friendly icon",
+  "36380780": "<p>Here's how to open a Touch/No Touch contract:</p><ul><li>Choose an instrument to trade.</li><li>Set the duration of your contract.</li><li>Set your preferred barrier.</li><li>Enter your stake or desired potential payout.</li><li>Place your trade.</li></ul>",
```

Comment:
Please make sure this dose not break anything because we have HTML elements inside our translations :thinking:

Reply:
These are for the SEO FAQ markup. Don't think it may break anything.